assets/setup-wizard/style.scss

```diff
@@ -225,19 +225,37 @@
 		max-width: 100%;
 
 		@media ( max-height: 500px ) {
-			.components-modal__header,
-			.sensei-setup-wizard__modal-footer {
-				position: static;
+			.components-modal__content {
+				.components-modal__header, .sensei-setup-wizard__modal-footer {
+					position: static;
+				}
 			}
 		}
+
 		.sensei-list__item:last-child {
 			border-bottom: 0;
 		}
 
+		&.components-modal__frame {
+			overflow: auto;
+			max-height: 90%;
```

Comment:
For the mentioned issue, I think we could fix that by adding it in a media query too?

Reply:
I think where we set the modal width too. Otherwise, in some screen sizes it's getting a blank space in the right too.

<img width="561" alt="Screen Shot 2022-06-15 at 15 21 14" src="https://user-images.githubusercontent.com/876340/173897803-551d1334-9a25-40c7-9ff4-5a4325af43c4.png">